src/utils/breakpoint/astBreakpointLocation.js

```diff
@@ -48,13 +48,13 @@ export function findClosestScope(functions: Scope[], location: Location) {
 
 export async function getASTLocation(
   source: Source,
+  symbols: Object,
```

Comment:
this might break other places `getASTLocation` is used, maybe we put it on the end
```js
export async function getASTLocation(
   source: Source, 
   location: Location, 
   symbols: Object): Promise<ASTLocation> {